src/components/Catalog.vue

```diff
@@ -890,4 +928,9 @@ td.provider .description {
   width: 100%;
   margin-bottom: 10px;
 }
+
+.external-pagination .page-link {
```

Comment:
Looks like this should be in ExternalPaginationLink.vue?

Reply:
oh right... and make it scoped so it doesn't spill (and doesn't need special class)